core/services/eth/models.go

```diff
@@ -171,6 +173,7 @@ func (h *Head) UnmarshalJSON(bs []byte) error {
 	h.Number = (*big.Int)(jsonHead.Number).Int64()
 	h.ParentHash = jsonHead.ParentHash
 	h.Timestamp = time.Unix(int64(jsonHead.Timestamp), 0).UTC()
+	h.BaseFeePerGas = (*utils.Big)((*big.Int)(jsonHead.BaseFeePerGas))
```

Comment:
Should be able to convert directly, no?
```suggestion
	h.BaseFeePerGas = (*utils.Big)(jsonHead.BaseFeePerGas)
```